apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx

```diff
@@ -21,67 +24,92 @@ export const MerchantMonitoringBusinessReport: FunctionComponent = () => {
     statusToBadgeData,
     tabs,
     activeTab,
+    notes,
+    isNotesOpen,
   } = useMerchantMonitoringBusinessReportLogic();
 
+  if (!notes || !businessReport) {
+    return;
+  }
```

Comment:
Not displaying the entire page considering the condition seems extreme. Is this supposed to be when loading? Without a loader or empty state it would look broken.